app/code/community/Bolt/Boltpay/controllers/Adminhtml/Sales/Order/CreateController.php

```diff
@@ -72,28 +72,36 @@ public function loadBlockAction()
         parent::loadBlockAction();
     }
 
+
     /**
      * Saving quote and create order.  We add the Bolt reference to the session
      */
     public function saveAction()
     {
-        // some versions of Magento store the shipping method at the top level of the $_POST array
```

Comment:
I don't think this removal is intentional?

Reply:
@daisy1754  yes. it was. please review:

https://github.com/BoltApp/bolt-magento1/blob/aed618e1498d86db325b3d56199f45e2a1617088/app/code/community/Bolt/Boltpay/controllers/Adminhtml/Sales/Order/CreateController.php#L191-L234